### DISCUSSION TITLE
Protokoll 15.12.24

### DISCUSSION THREAD

**Discussion by anomatomato** [AuDojo/AuDojo#132] [Protokolle]
## Was haben wir gemacht?

- Tutorial wenn SortSensei geöffnet wird
- Next/prev button
- Emojis für Buttons
- Paar Bugs gefunden und in Issues geschrieben

## Was machen wir jetzt?

- Minseo: HTML/CSS SuperSimpleDev, Paar für BubbleSort zurückgeben
- Trang: D3 Tutorial
- An: D3 Tutorial, Tabelle für SelectionSort und BubbleSort vervollständigen, Bugs beheben
- Maximo: E-Mail, Login (HTML/CSS Tutorial)
- Jan: Datenbank, Mehr Frontend

**Comment by jac0mp** [comment]
Arbeite neben den Issues gerade auch ein weiteres React-Tutorial durch. 
Habe gemerkt dass ich Probleme damit habe code umzustrukturieren / sinnvoll zu ergänzen
